website/src/controllers/groups.js

```diff
@@ -137,7 +137,28 @@ api.get = function(req, res, next) {
   q.exec(function(err, group){
     if (err) return next(err);
     if (!group && gid!=='party') return res.json(404,{err: "Group not found or you don't have access."});
```

Comment:
Weird. :P I didn't change this line, I think. So, why did this start breaking things? Or did it just break the tests?